vitess.io/_includes/left-nav-menu.html

```diff
@@ -38,12 +38,18 @@ <h4 class="arrow-r">Reference Guides</h4>
         <li class="submenu">
           <h4 class="arrow-r">Other Resources</h4>
           <ul style="display: none">
-            <li><a href="/resources/presentations.html">Presentations</a>
+            <li><a href="/resources/presentations.html">Presentations</a></li>
+            <li style="padding-bottom: 0"><a href="/posts/">Blog</a>
+              <ul style="display: block">
+                <li><a href="/cloud-native-mysql-sharding/">Cloud Native MySQL Sharding</a></li>
+                <li><a href="/scaling-mysql-in-cloud-with-vitess-and-kubernetes/">Scaling MySQL in the Cloud</a></li>
```

Comment:
As we write more posts, this won't scale (pun intended). I think we should just link to the top-level Blog in the side menu.